lib/system/web/src/lib.rs

```diff
@@ -244,6 +259,26 @@ impl NodeAppender for Node {
                 )
             );
     }
+
+    fn prepend_or_panic(&self, node : &Node) {
```

Comment:
The naming is inconsistent. We've got `append_CHILD_or_panic` and `prepend_or_panic`.

Reply:
Renamed to:

```rust
    fn append_or_panic(&self, node : &Node);
    fn prepend_or_panic(&self, node : &Node);
    fn insert_before_or_panic(&self, node : &Node, reference_node : &Node);
```